cmd/new.go

```diff
@@ -168,22 +168,6 @@ func runNew(args newArgs) error {
 		}
 	}
 
-	// If a stack was specified via --stack, see if it already exists.
-	var s backend.Stack
-	if args.stack != "" {
-		existingStack, existingName, existingDesc, err := getStack(args.stack, opts)
```

Comment:
Hmm, so the web app creates a stack and CLI takes it over? How does this interplay with the requirement of a unique project? I guess it does currently because we don't check the project name from the stack name, which was one the issues to be fixed.